README.md

```diff
@@ -14,3 +14,7 @@ For any correction or amendment to the value, create a fork, make a branch with
 
 ## Values
 
+### Delivery 
```

Comment:
Issue: this one got mixed I believe got the wrong changes as is equal than #6

Reply:
Good spot, this one should be the communication value

Reply:
Replaced with communication value, @octavioamu you want to take another look 🙂 

_If you want to keep the 'request changes' review flag, you'll want to add a another blocking comment_